tests/contrib/test_lint.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
I believe that what we wanted to test is that the module is importable in Python 3.
So testing it just requires to import it in a test file _at least_.
(theorically, if you try the test without the fix you'll see a failing test showing the issue — in Python 3).

Writing more tests would be part of another pull request to improve coverage of this middleware.